apps/desktop-wallet/src/api/addressesFungibleTokensPricesDataHooks.tsx

```diff
@@ -98,14 +102,32 @@ export const useSortTokensByWorth = (tokens: Asset[]) => {
   )
 }
 
-export const useAddressesTokensWorth = (addressHashes?: AddressHash[] | AddressHash) => {
-  const { data: tokenPrices } = useAddressesTokensPrices()
-  const selectAddressesKnownFungibleTokens = useMemo(makeSelectAddressesKnownFungibleTokens, [])
-  const tokens = useAppSelector((s) => selectAddressesKnownFungibleTokens(s, addressHashes))
+export const useAddressesTokensWorth = (addressHash?: AddressHash) => {
```

Comment:
Ok so, if the `addressHash` is omitted, this will return the list of token worth for all addresses, correct? 
I don't know if I like that or not 🤔 
I probably need to think more about this and we could discuss this, but intuitively I don't find this super explicit.

Reply:
We have already been using this pattern a lot in the current codebase. You can look for it in components that both work in the overview page as well as in address-specific pages/modals. It's a simple way to achieve something that otherwise is complicated. Passing an array of `[addressHash]` always creates a new reference to an object. Having a `string | undefined` instead is simpler and more efficient. Happy to discuss alternative naming.

Reply:
Yeah, the alternative would be to create multiple functions. `useAddressTokensWorth(string)`, `useAddressesTokensWorth([string])`, `useAllAddressesTokensWorth`. I went that way in my Tanstack branch: creating a base hook which is reused in other smol more specific hooks. 

But again, I'm not yet sure which option is the best.